src/boilerplate/contract/solidity/nodes/ContractBoilerplateGenerator.ts

```diff
@@ -118,7 +118,7 @@ class ContractBoilerplateGenerator {
       let paramtype: string;
       let params : any[];
       let functionName: string;
-      
+      console.log('circuit', circuitParams);
```

Comment:
This this is for debugging?

Reply:
yes thanks.. I will remove all the consoles